kerl

```diff
@@ -2215,6 +2230,38 @@ case "$1" in
                 ;;
         esac
         ;;
+    upgrade)
+        if [ $# -ne 1 ]; then
+            upgrade_usage
+            exit 1
+        fi
+        current_kerl_path="$(which kerl)"
+        which_status=$?
+        if [ $which_status != 0 ]; then
+            if [ -z ${KERL_APP_INSTALL_DIR+x} ]; then
```

Comment:
@jadeallenx, I ended up "borrowing" the logic from https://github.com/kerl/kerl/blob/master/kerl#L2158. Also, I added a small update to the `README.md` so this isn't a hidden option.